.github/workflows/scheduled-class-upload.yml

```
@@ -32,7 +32,27 @@ jobs:
 
       - name: 'Upload classes from primary vimeo account'
         run: npm run tools:classes:last
-        
+
+      - name: Slack Notification
+        id: slack
+        if: ${{ failure() }}
+        uses: slackapi/slack-github-action@v1.19.0
+        with: 
+          channel-id: 'C01GKL43JRY'
+          slack-message: ':boom: Upload classes ${{ job.status }}'
```

Podemos ocupar aqui :alert: en lugar de boom porfa? 
```suggestion
          slack-message: ':alert: Upload classes ${{ job.status }}'
```

---

Ok. Estaba demasiado catastrofico, lo cambio.